app/tests/beaker-plugin-safe-app/.gitignore

```diff
@@ -0,0 +1,2 @@
+build
+yarn.lock
```

Comment:
We shouldn't be ignoring `yarn.lock` files: https://yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control